counterexamples/common/timestamp/bad-timestamp-invalid-days.yaml

```diff
@@ -0,0 +1,15 @@
+---
+id: bad-timestamp-invalid-days
+type: Feature
+geometry:
+  type: Point
+  coordinates: [0, 0]
+properties:
+  categories:
+    main: someCategory
+  theme: places
+  type: place
+  version: 0
+  updateTime: "2024-02-32T15:45:22Z"
+  extExpectedErrors:
+    - "does not match pattern"
```

Comment:
Can you fix up your editor so it adds newlines at end of file?

Whether using IntelliJ or VS Code or whatever, there's usually a setting you can click to do this.

Reply:
Fixed in this and all other files